### ISSUE TITLE
Add cyclic_value constructor: Initialize with specific value

### ISSUE THREAD

**Issue by Sn1cket** [ETLCPP/etl#662]
Hi,
I'm using the cyclic_value in my project and in most cases I wanted to assign an initial value other than the FIRST value of the class. So I was wondering if it would make sense to add an additional constructor that could handle that case.

Maybe something like this:
````
//*************************************************************************
/// Constructor.
/// The initial value is set to a specfic value.
//*************************************************************************
cyclic_value(T t)
{
  set(t);
}
````

Best regards

**Comment by jwellbelove** [comment]
Yes, that seems reasonable.

**Comment by jwellbelove** [comment]
Fixed 20.35.12
